### PR TITLE
Form Component

### PR DESCRIPTION
**Pull request by sejaljain123**
#7 Form Component
Sign in and Register  Form
sub-issue of #2

### CODE REVIEW

File: src/components/Footer/Footer.scss [hunk 1]
```diff
@@ -21,5 +22,6 @@
 @media screen and (min-width: 0px) and (max-width: 400px) {
```
Comment (manishprivet) [inline src/components/Footer/Footer.scss:22]: We don't need the min-width property here as keeping it as 0 is useless.

File: src/components/Footer/Footer.scss [hunk 2]
```diff
@@ -1,5 +1,6 @@
 .footer {
   position: absolute;
+
```
Comment (manishprivet) [inline src/components/Footer/Footer.scss:3]: Unused whitespaces should be avoided.

**Review by manishprivet — CHANGES_REQUESTED** [review]

File: src/components/Footer/Footer.scss (hunk 1 above)
Comment (sejaljain123) [reply src/components/Footer/Footer.scss:21]: Got it!

File: src/components/Footer/Footer.scss (hunk 2 above)
Comment (sejaljain123) [reply src/components/Footer/Footer.scss]: okay 😅

**Review by manishprivet — APPROVED** [review]

**Pull request merged**